capstone/home.html

```diff
@@ -0,0 +1,134 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <title>Rock & Roll Hall of Fame</title>
+</head>
+<body>
+  <header>
+    <div class="topBar">
+      <i class="fa-brands fa-facebook-f"></i>
+      <i class="fa-brands fa-twitter"></i>
+      <a href="#">English</a>
+      <a href="#">My page</a>
+      <a href="#">Logout</a>
+    </div>
+    <nav>
+    <div class="navbar">
+      <a class="logo" href="home.html" ><img src="images/logo.svg"></a>
+      <div class="nav-items">
+        <a class="aboutPage" href="about.html">About</a>
+        <a href="home.html#program">Program</a>
+        <a href="home.html#join">Join</a>
+        <a href="#sponsor">Sponsor</a>
+        <a href="about.html#news">News</a>
+        <a class="rockHall" href="#rockhall">Rock Hall</a>
+      </div>
+    </div>
+    </nav>
+    <div class="menu">
+      <a class="menu-button open" href="#"><i class="fa-solid fa-bars"></i></a>
+      <a class="menu-button close" href="#"><i class="fa-solid fa-xmark"></i></a>
+      <div hidden class="menu-content">
+        <a class="logo" href="home.html" ><img src="images/logo.svg"></a>
+        <div class="menu-items">
+          <a class="aboutPage" href="about.html">About</a>
+          <a href="home.html#program">Program</a>
+          <a href="home.html#join">Join</a>
+          <a href="#sponsor">Sponsor</a>
+          <a href="about.html#news">News</a>
+        </div>
+      </div>
+    </div>
+  </header>
+  <main>
+    <div class="main">
+      <div class="homeContent home">
+        <h2 class="greet">"Hello! Rock & Roll Fans"</h2>
+        <h1 class="title">ROCK & ROLL HALL OF FAME 2023</h1>
+        <p class="description">The ROCK & ROLL Hall of Fame celebrates the sound of youth culture and honors the artists whose music connects us all. Join us in our next presentations in April, Cleveland.</p>
+        <h2 class="date">2023.04.27(THU) - 29(SAT)</h2>
+        <h3 class="location">@ The Rock and Roll Hall of Fame Museum, Cleveland</h3>
+      </div>
+    </div>
+    <div id="program" class="mainProgram">
+      <div class="holder">
+        <h2 class="title">Main Program</h2>
+        <div class="underLine"></div>
+      </div>
+      <div class="programElements">
+        <div class="elements">
+          <i class="fa-sharp fa-solid fa-hourglass-start"></i>
+          <h3>Tour starts</h3>
+          <p>It's a long way to the top if you want to rock & roll, sometimes you have to start at the bottom. </p>
+        </div>
+        <div class="elements">
+          <i class="fa-solid fa-guitar"></i>
+          <h3>The Garage</h3>
+          <p>We're getting the band back together! The Garage is fully opened and ready to rock.</p>
+        </div>
+        <div class="elements">
+          <i class="fa-solid fa-star"></i>
```

Comment:
- [ ] You did a great job here, but kindly try to use the same background image here to be like the Behance and with that, it will be nice 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/217924326-d6cdee25-9084-40c5-9ebb-ccfacdc6894b.png)   |   ![image](https://user-images.githubusercontent.com/83360636/217924278-f1103af2-aab4-4d1c-a7d7-248e8aa16b83.png)  |